new-log-viewer/src/App.tsx

```diff
@@ -1,7 +1,91 @@
+import {
+    CssVarsProvider,
+    extendTheme,
+} from "@mui/joy/styles";
+
+import KeyboardArrowDown from "@mui/icons-material/KeyboardArrowDown";
+
 import Layout from "./components/Layout";
 import StateContextProvider from "./contexts/StateContextProvider";
 import UrlContextProvider from "./contexts/UrlContextProvider";
+import {CONFIG_KEY} from "./typings/config";
+import {CONFIG_DEFAULT} from "./utils/config";
+
 
+const monacoTheme = extendTheme({
+    colorSchemes: {
+        light: {
+            palette: {
+                success: {
+                    solidBg: "#007acc",
+                    solidHoverBg: "#0062a3",
+                    solidActiveBg: "#0062a3",
+                },
+                neutral: {
+                    solidBg: "#5f6a79",
+                    solidHoverBg: "#4c5561",
+                    solidActiveBg: "#4c5561",
+                },
+                focusVisible: "#0090f1",
+            },
+        },
+        dark: {
+            palette: {
+                success: {
+                    solidBg: "#0e639c",
+                    solidHoverBg: "#1177bb",
+                    solidActiveBg: "#1177bb",
+                },
+                neutral: {
+                    solidBg: "#313131",
+                    solidHoverBg: "#45494e",
+                    solidActiveBg: "#45494e",
+                },
+                focusVisible: "#007fd4",
+            },
+        },
+    },
+    focus: {
+        default: {
+            outlineWidth: "3px",
+        },
+    },
+    fontFamily: {
+        body: "-apple-system, system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif",
```

Comment:
Note they Microsoft fonts' licenses might not be compatible with yscope-log-viewer's license and we might need licenses to be able to integrate the Microsoft fonts when we want to derive a commercial product from yscope-log-viewer. 

For now, I think it's safe to keep `-apple-system, BlinkMacSystemFont, system-ui, Ubuntu, "Droid Sans", Roboto`. We will introduce a global style variable `--ylv-ui-font-family` in #56 which you might want to leverage after that PR is merged.